ap/settings/common.py

```diff
@@ -81,6 +82,29 @@
     "django_structlog.middlewares.RequestMiddleware",
 ]
 
+# -- Sentry error tracking
+
+if os.environ.get("SENTRY_DSN"):
+    SENTRY_ENVIRONMENT = ENV
```

Comment:
Not sure there is any point to this either, just use `ENV` below?

Reply:
Sorted